standalonecontroller/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-standalonecontroller"
+description: "Red Hat JBoss BPM Suite 7.0 Standalone Controller container image"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite-standalonecontroller"
+    - name: "JBOSS_BPMSUITE_VERSION"
```

Comment:
Thanks @rbarriuso . Please let us know when it's ready so this PR can get processed. :)